app/Http/Controllers/SettingsController.php

```diff
@@ -186,7 +173,7 @@ public function delete(Request $request)
             }
         }
 
-        DB::table('accounts')->where('id', $account->id)->delete();
+        $account->delete();
```

Comment:
Perfect!
This should be done on the `reset` function too?

Reply:
It’s not possible, because in the case of a reset, we don't delete the account. So I can't evoke this and make it happen. I don't see how we can do it simply without iterating on every table like we do now.

Reply:
We could delete all contacts for this account, and this should clean everything?

Reply:
You are right. I’m going to write a service for this.